Purchases/Public/RCPurchases.m

```diff
@@ -561,41 +611,38 @@ - (void)updateOriginalApplicationVersion
 
 - (void)createAlias:(NSString *)alias
 {
-    [self createAlias:alias completion:^(NSError * _Nullable error) {}];
+    [self createAlias:alias completionBlock:^(RCPurchaserInfo *info, NSError *e) {}];
 }
 
-- (void)createAlias:(NSString *)alias completion:(void (^)(NSError * _Nullable error))completion
+- (void)createAlias:(NSString *)alias completionBlock:(RCReceivePurchaserInfoBlock)completion
 {
     [self.backend createAliasForAppUserID:self.appUserID withNewAppUserID:alias completion:^(NSError * _Nullable error) {
```

Comment:
To avoid making another API call, the alias server call could return the new purchaser info. We could cache it so that the identify function can call the completion block with the newly cached version of purchaser info.